website/source/docs/platform/k8s/helm/configuration.html.md

```diff
@@ -31,6 +31,48 @@ and consider if they're appropriate for your deployment.
 
     * `tlsDisable` (`boolean: true`) - When set to `true`, changes URLs from `https` to `http` (such as the `VAULT_ADDR=http://127.0.0.1:8200` environment variable set on the Vault pods).
 
+* `injector` - Values that configure running a Vault Agent Injector Admission Webhook Controller within Kubernetes.
+    
+    * `enabled` (`boolean: true`) - When set to `true`, the Vault Agent Injector Admission Webhook controller will be created.
```

Comment:
Question, does `boolean:true` denote that it defaults to true?

Reply:
Yes!